gormigrate.go

```diff
@@ -294,6 +299,20 @@ func (g *Gormigrate) RollbackTo(migrationID string) error {
 	return g.commit()
 }
 
+func (g *Gormigrate) GetLastRunMigrationID() (string, error) {
+	mig, err := g.GetLastRunMigration()
+	// Check error
+	if err != nil {
+		return "", err
+	}
+
+	return mig.ID, nil
+}
+
+func (g *Gormigrate) GetLastRunMigration() (*Migration, error) {
+	return g.getLastRunMigration()
```

Comment:
This causes a nil pointer dereference because it eventually calls `migrationRan`, which needs there to be an active transaction ongoing, but `begin()` is never called so `g.tx` is nil.

Reply:
Good catch ! Should be ok now no ?

Reply:
Yep, thanks!